test/cli.js

```diff
@@ -31,3 +31,9 @@ test.skip('ignores fixture', async t => {
 	const cwd = path.join(__dirname, 'fixtures/ignores');
 	t.throws(execa('../../../cli.js', ['--no-local'], {cwd}));
 });
+
+test('supports shareable config', async () => {
+	await execa('rm', ['-rf', '../node_modules/xo'], {cwd: __dirname});
```

Comment:
Oh yes, I guess this is another, separate issue. https://github.com/sindresorhus/xo/blob/master/package.json#L91 causes `cli.js` to stop execution and rely on `node_modules/xo` rather than version in `master` branch. So when you're testing `cli.js`, you are not actually testing it, because it spawns `cli.js` from `node_modules/xo`.